skyportal/tests/frontend/test_sources.py

```diff
@@ -299,32 +299,27 @@ def test_super_user_can_delete_unowned_comment(
     driver.wait_for_xpath(f'//div[text()="{public_source.id}"]')
     comment_box = driver.wait_for_xpath("//input[@name='text']")
     comment_text = str(uuid.uuid4())
+    driver.scroll_to_element_and_click(comment_box)
     comment_box.send_keys(comment_text)
     driver.scroll_to_element_and_click(
-        driver.find_element_by_xpath('//*[@name="submitCommentButton"]')
+        driver.wait_for_xpath('//*[@name="submitCommentButton"]')
     )
-    try:
-        comment_text_div = driver.wait_for_xpath(f'//div[./p[text()="{comment_text}"]]')
-    except TimeoutException:
-        driver.refresh()
-        comment_text_div = driver.wait_for_xpath(f'//div[./p[text()="{comment_text}"]]')
+
     driver.get(f"/become_user/{super_admin_user.id}")
     driver.get(f"/source/{public_source.id}")
-    driver.refresh()
+
     comment_text_div = driver.wait_for_xpath(f'//div[./p[text()="{comment_text}"]]')
     comment_div = comment_text_div.find_element_by_xpath("..")
     comment_id = comment_div.get_attribute("name").split("commentDiv")[-1]
-    delete_button = comment_div.find_element_by_xpath(
+
+    # wait for delete button to become interactible - hence pause 0.1
+    ActionChains(driver).move_to_element(comment_text_div).pause(0.1).perform()
```

Comment:
for this, we need to do a manual actionchain movetoelement, as the delete button is only interactible if the cursor is hovering directly over the comment div. If we use a straight up `click_xpath` or `scroll_to_element_and_click` here then it will fail with an ElementNotINteractibleException such as 

https://github.com/skyportal/skyportal/runs/1256617595